README.md

```diff
@@ -599,9 +599,12 @@ Some companies require a special attribution notice. View a list of the companie
 - IBM&reg;
 - IBM Cloud&reg;
 - IBM Quantum&trade;
-- Qiskit&reg;
 </details>
 
+<Admonition type="note" title="Qiskit is an exception">
```

Comment:
This doesn't work in GitHub's README, which doesn't have MDX components.

Reply:
haha oops! I'll remove those